src/main/java/org/codehaus/mojo/versions/api/DefaultVersionsHelper.java

```diff
@@ -215,6 +218,13 @@ public DefaultVersionsHelper( RepositorySystem repositorySystem, ArtifactResolve
         this.log = log;
     }
 
+    /**
+     * Private constructor used by the builder
+     */
+    private DefaultVersionsHelper()
```

Comment:
One constructor and everything should be done in builder

Reply:
will do